apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -52,6 +53,9 @@ function WpcomNux() {
 	if ( ! isWpcomNuxEnabled || isSPTOpen ) {
 		return null;
 	}
+	recordTracksEvent( 'calypso_newsite_wpcomnux_view', {
+		is_new_flow: isGutenboarding,
```

Comment:
I think `isGutenboarding` is defined a few lines beneath, so this will be `undefined`